src/screens/OptionsScreen.js

```diff
@@ -4,7 +4,7 @@ import { StyleSheet, ScrollView, Text, View } from 'react-native';
 import { CheckBox, Button } from 'react-native-elements';
 
 import theme from '../utils/theme.js';
-import questions from '../../data/questions';
+import questions from '../../data/questions_EN_US';
```

Comment:
The app should handle it for you now :)

Reply:
closing in favor of https://github.com/sallesma/timeout-ultimate/pull/56 which does not require a rebase